tests/_support/Commands/LanguageCommand.php

```diff
@@ -1,37 +0,0 @@
-<?php
-
-namespace Tests\Support\Commands;
-
-use CodeIgniter\CLI\GeneratorCommand;
-
-class LanguageCommand extends GeneratorCommand
```

Comment:
Reinstate this test file for the configurable sort imports test. You cannot just delete a test just to make the tests succeed.